EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,219 @@
+---
+eip:
```

Comment:
```suggestion
eip: 7822
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.